src/FormTest.tsx

```diff
@@ -2,7 +2,7 @@
 // import { CustomFields } from '../legos/config';
 // import { TABULA_FORMS } from '../legos/form';
 
-import { FormBuilder } from '@daohaus/form-builder';
+import { FormBuilder } from '@daohaus/haus-form-builder';
```

Comment:
name is in package.json, that beign said, i think we should change it to `@daohaus/form-builder` in the monorepo